docs/07-sales-and-marketing/services.md

```diff
@@ -0,0 +1,57 @@
+# Services (WIP)
+
+## Application Migration
+* Drupal Application Migration
+
+## Cloud-Based Infrastructure
+* Systems Administration (SA)
+* Hosting Services
+
+## Design
+* User Research
+* Content Strategy
+* Information Architecture
+* Interaction Design and Front End Development
+* Visual Design
+
+## Development
```

Comment:
I've long since been torn about backend and frontend development (should they be separated?)

What do you think? At a minimum, we need to mention competency in theming (visual systems) and JS-based Frameworks (Angular, Ember, React, etc)